usage/app/controllers/UsageApi.scala

```diff
@@ -86,6 +91,34 @@ object UsageApi extends Controller with ArgoHelpers {
 
   }
 
+  def reindexForContent(contentId: String) = Authenticated.async {
+    val query = ItemQuery(contentId)
+      .showFields("all")
+      .showElements("all")
+
+    val result = LiveContentApi.getResponse(query).map(response => {
+      response.content match {
+        case Some(content) => {
+          val contentFirstPublished =
+            LiveContentApi.getContentFirstPublished(content)
+          val container = contentFirstPublished
+            .map(new LiveContentItem(content, _))
+            .map(_.copy(isReindex = true))
+
+          container.map(LiveCrierContentStream.observable.onNext(_))
+        }
+        case _ => Unit
+      }
+    })
+
+    result
+      .map(_ => Accepted)
+      .recover { case error: Exception => {
+        Logger.error("UsageApi reindex for for content failed!", error)
```

Comment:
Will this log the content ID? Might help for debugging and rechecking usage later?